olive/passes/pytorch/lora.py

```diff
@@ -964,7 +964,7 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
                 description="Number of LoftQ iterations.",
             ),
         }
-        config.update(QLoRABase._default_config(accelerator_spec))  # pylint: disable=protected-access
+        config.update(super()._default_config(accelerator_spec))
```

Comment:
@jambayk, this is a behavior change, could you help confirm?

Reply:
This should not have been a behavior change. But currently it is because I mistakenly made the parent of this class as `QLoRA` when it was supposed to be `QLoRABase`. Can you make that correct that as well? Thanks!

Reply:
with the new super(), its base class will be QLoRA

Reply:
> This should not have been a behavior change. But currently it is because I mistakenly made the parent of this class as `QLoRA` when it was supposed to be `QLoRABase`. Can you make that correct that as well? Thanks!

Or do you mean we need change the base class as QLoRABase instead of QLoRA?

Reply:
Yes, update the base class for `LoftQ` as `QLoRABase` since it was a mistake from my side. You can see that all the methods defined in `QLoRA` are overwritten in `LoftQ`.

Reply:
done